### PR TITLE
chore: create preview version of react-tree

### PR DESCRIPTION
**Pull request by bsunderhus**
## New Behavior

1. creates `react-tree-preview` package as copy of `react-tree`

## Related Issue(s)

- Fixes #

### CODE REVIEW

**Comment by fabricteam** [comment]
## 📊 Bundle size report

<sub>🤖 This report was generated against <a href='https://github.com/microsoft/fluentui/commit/d5ece238df797e84a59affc6162b0fa5cc46649d'>d5ece238df797e84a59affc6162b0fa5cc46649d</a></sub>
<div id="bundleSizeComment9423"/>

**Comment by fabricteam** [comment]
## [Perf Analysis (`@fluentui/react-components`)](https://github.com/microsoft/fluentui/wiki/Perf-Testing)
<p>No significant results to display.</p><details><summary>All results</summary><p>
  <table>
  <tr>
    <th>Scenario</th>
    <th>Render type</th>
    <th>
      <a href="https://github.com/microsoft/fluentui/wiki/Perf-Testing#why-are-results-listed-in-ticks-instead-of-time-units">Master Ticks</a>
    </th>
    <th>
      <a href="https://github.com/microsoft/fluentui/wiki/Perf-Testing#why-are-results-listed-in-ticks-instead-of-time-units">PR Ticks</a>
    </th>
    <th>Iterations</th>
    <th>Status</th>
  </tr><tr>
            <td>Avatar</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/28760/perf-test-react-components/Avatar-mount_base.html">649</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/28760/perf-test-react-components/Avatar-mount.html">645</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>Button</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/28760/perf-test-react-components/Button-mount_base.html">315</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/28760/perf-test-react-components/Button-mount.html">306</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>Field</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/28760/perf-test-react-components/Field-mount_base.html">1178</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/28760/perf-test-react-components/Field-mount.html">1153</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>FluentProvider</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/28760/perf-test-react-components/FluentProvider-mount_base.html">668</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/28760/perf-test-react-components/FluentProvider-mount.html">683</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>FluentProviderWithTheme</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/28760/perf-test-react-components/FluentProviderWithTheme-mount_base.html">73</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/28760/perf-test-react-components/FluentProviderWithTheme-mount.html">80</a></td>
            <td>10</td>
            <td></td>
           </tr>
<tr>
            <td>FluentProviderWithTheme</td>
            <td>virtual-rerender</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/28760/perf-test-react-components/FluentProviderWithTheme-virtual-rerender_base.html">66</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/28760/perf-test-react-components/FluentProviderWithTheme-virtual-rerender.html">74</a></td>
            <td>10</td>
            <td></td>
           </tr>
<tr>
            <td>FluentProviderWithTheme</td>
            <td>virtual-rerender-with-unmount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/28760/perf-test-react-components/FluentProviderWithTheme-virtual-rerender-with-unmount_base.html">76</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/28760/perf-test-react-components/FluentProviderWithTheme-virtual-rerender-with-unmount.html">76</a></td>
            <td>10</td>
            <td></td>
           </tr>
<tr>
            <td>InfoButton</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/28760/perf-test-react-components/InfoButton-mount_base.html">5</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/28760/perf-test-react-components/InfoButton-mount.html">15</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>MakeStyles</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/28760/perf-test-react-components/MakeStyles-mount_base.html">837</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/28760/perf-test-react-components/MakeStyles-mount.html">876</a></td>
            <td>50000</td>
            <td></td>
           </tr>
<tr>
            <td>Persona</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/28760/perf-test-react-components/Persona-mount_base.html">1774</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/28760/perf-test-react-components/Persona-mount.html">1737</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>SpinButton</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/28760/perf-test-react-components/SpinButton-mount_base.html">1422</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/28760/perf-test-react-components/SpinButton-mount.html">1427</a></td>
            <td>5000</td>
            <td></td>
           </tr></table></p></details>

<div id="perfComment9425"/>

**Comment by fabricteam** [comment]
## Perf Analysis (`@fluentui/react-northstar`)
<details><summary>Perf tests with no regressions</summary>

Scenario | Current PR Ticks | Baseline Ticks | Ratio
:--- | ---:| ---:| ---:
ChatWithPopoverPerf.default | [196](https://fluentuipr.z22.web.core.windows.net/pull/28760/perf-test-northstar/ChatWithPopoverPerf.default.html) | [178](https://fluentuipr.z22.web.core.windows.net/pull/28760/perf-test-northstar/ChatWithPopoverPerf.default_base.html) | 1.1:1
RadioGroupMinimalPerf.default | [267](https://fluentuipr.z22.web.core.windows.net/pull/28760/perf-test-northstar/RadioGroupMinimalPerf.default.html) | [247](https://fluentuipr.z22.web.core.windows.net/pull/28760/perf-test-northstar/RadioGroupMinimalPerf.default_base.html) | 1.08:1
RefMinimalPerf.default | [116](https://fluentuipr.z22.web.core.windows.net/pull/28760/perf-test-northstar/RefMinimalPerf.default.html) | [107](https://fluentuipr.z22.web.core.windows.net/pull/28760/perf-test-northstar/RefMinimalPerf.default_base.html) | 1.08:1
TableMinimalPerf.default | [242](https://fluentuipr.z22.web.core.windows.net/pull/28760/perf-test-northstar/TableMinimalPerf.default.html) | [227](https://fluentuipr.z22.web.core.windows.net/pull/28760/perf-test-northstar/TableMinimalPerf.default_base.html) | 1.07:1
BoxMinimalPerf.default | [193](https://fluentuipr.z22.web.core.windows.net/pull/28760/perf-test-northstar/BoxMinimalPerf.default.html) | [182](https://fluentuipr.z22.web.core.windows.net/pull/28760/perf-test-northstar/BoxMinimalPerf.default_base.html) | 1.06:1
DividerMinimalPerf.default | [206](https://fluentuipr.z22.web.core.windows.net/pull/28760/perf-test-northstar/DividerMinimalPerf.default.html) | [194](https://fluentuipr.z22.web.core.windows.net/pull/28760/perf-test-northstar/DividerMinimalPerf.default_base.html) | 1.06:1
FlexMinimalPerf.default | [165](https://fluentuipr.z22.web.core.windows.net/pull/28760/perf-test-northstar/FlexMinimalPerf.default.html) | [156](https://fluentuipr.z22.web.core.windows.net/pull/28760/perf-test-northstar/FlexMinimalPerf.default_base.html) | 1.06:1
ReactionMinimalPerf.default | [214](https://fluentuipr.z22.web.core.windows.net/pull/28760/perf-test-northstar/ReactionMinimalPerf.default.html) | [201](https://fluentuipr.z22.web.core.windows.net/pull/28760/perf-test-northstar/ReactionMinimalPerf.default_base.html) | 1.06:1
ImageMinimalPerf.default | [225](https://fluentuipr.z22.web.core.windows.net/pull/28760/perf-test-northstar/ImageMinimalPerf.default.html) | [215](https://fluentuipr.z22.web.core.windows.net/pull/28760/perf-test-northstar/ImageMinimalPerf.default_base.html) | 1.05:1
SegmentMinimalPerf.default | [199](https://fluentuipr.z22.web.core.windows.net/pull/28760/perf-test-northstar/SegmentMinimalPerf.default.html) | [190](https://fluentuipr.z22.web.core.windows.net/pull/28760/perf-test-northstar/SegmentMinimalPerf.default_base.html) | 1.05:1
TreeWith60ListItems.default | [89](https://fluentuipr.z22.web.core.windows.net/pull/28760/perf-test-northstar/TreeWith60ListItems.default.html) | [85](https://fluentuipr.z22.web.core.windows.net/pull/28760/perf-test-northstar/TreeWith60ListItems.default_base.html) | 1.05:1
AlertMinimalPerf.default | [161](https://fluentuipr.z22.web.core.windows.net/pull/28760/perf-test-northstar/AlertMinimalPerf.default.html) | [155](https://fluentuipr.z22.web.core.windows.net/pull/28760/perf-test-northstar/AlertMinimalPerf.default_base.html) | 1.04:1
AttachmentMinimalPerf.default | [88](https://fluentuipr.z22.web.core.windows.net/pull/28760/perf-test-northstar/AttachmentMinimalPerf.default.html) | [85](https://fluentuipr.z22.web.core.windows.net/pull/28760/perf-test-northstar/AttachmentMinimalPerf.default_base.html) | 1.04:1
ListMinimalPerf.default | [307](https://fluentuipr.z22.web.core.windows.net/pull/28760/perf-test-northstar/ListMinimalPerf.default.html) | [296](https://fluentuipr.z22.web.core.windows.net/pull/28760/perf-test-northstar/ListMinimalPerf.default_base.html) | 1.04:1
ProviderMergeThemesPerf.default | [682](https://fluentuipr.z22.web.core.windows.net/pull/28760/perf-test-northstar/ProviderMergeThemesPerf.default.html) | [658](https://fluentuipr.z22.web.core.windows.net/pull/28760/perf-test-northstar/ProviderMergeThemesPerf.default_base.html) | 1.04:1
AvatarMinimalPerf.default | [111](https://fluentuipr.z22.web.core.windows.net/pull/28760/perf-test-northstar/AvatarMinimalPerf.default.html) | [108](https://fluentuipr.z22.web.core.windows.net/pull/28760/perf-test-northstar/AvatarMinimalPerf.default_base.html) | 1.03:1
ButtonSlotsPerf.default | [308](https://fluentuipr.z22.web.core.windows.net/pull/28760/perf-test-northstar/ButtonSlotsPerf.default.html) | [298](https://fluentuipr.z22.web.core.windows.net/pull/28760/perf-test-northstar/ButtonSlotsPerf.default_base.html) | 1.03:1
ChatDuplicateMessagesPerf.default | [157](https://fluentuipr.z22.web.core.windows.net/pull/28760/perf-test-northstar/ChatDuplicateMessagesPerf.default.html) | [152](https://fluentuipr.z22.web.core.windows.net/pull/28760/perf-test-northstar/ChatDuplicateMessagesPerf.default_base.html) | 1.03:1
ChatMinimalPerf.default | [428](https://fluentuipr.z22.web.core.windows.net/pull/28760/perf-test-northstar/ChatMinimalPerf.default.html) | [416](https://fluentuipr.z22.web.core.windows.net/pull/28760/perf-test-northstar/ChatMinimalPerf.default_base.html) | 1.03:1
HeaderSlotsPerf.default | [471](https://fluentuipr.z22.web.core.windows.net/pull/28760/perf-test-northstar/HeaderSlotsPerf.default.html) | [458](https://fluentuipr.z22.web.core.windows.net/pull/28760/perf-test-northstar/HeaderSlotsPerf.default_base.html) | 1.03:1
ItemLayoutMinimalPerf.default | [708](https://fluentuipr.z22.web.core.windows.net/pull/28760/perf-test-northstar/ItemLayoutMinimalPerf.default.html) | [685](https://fluentuipr.z22.web.core.windows.net/pull/28760/perf-test-northstar/ItemLayoutMinimalPerf.default_base.html) | 1.03:1
ListNestedPerf.default | [320](https://fluentuipr.z22.web.core.windows.net/pull/28760/perf-test-northstar/ListNestedPerf.default.html) | [310](https://fluentuipr.z22.web.core.windows.net/pull/28760/perf-test-northstar/ListNestedPerf.default_base.html) | 1.03:1
ListWith60ListItems.default | [365](https://fluentuipr.z22.web.core.windows.net/pull/28760/perf-test-northstar/ListWith60ListItems.default.html) | [354](https://fluentuipr.z22.web.core.windows.net/pull/28760/perf-test-northstar/ListWith60ListItems.default_base.html) | 1.03:1
MenuButtonMinimalPerf.default | [967](https://fluentuipr.z22.web.core.windows.net/pull/28760/perf-test-northstar/MenuButtonMinimalPerf.default.html) | [940](https://fluentuipr.z22.web.core.windows.net/pull/28760/perf-test-northstar/MenuButtonMinimalPerf.default_base.html) | 1.03:1
VideoMinimalPerf.default | [437](https://fluentuipr.z22.web.core.windows.net/pull/28760/perf-test-northstar/VideoMinimalPerf.default.html) | [425](https://fluentuipr.z22.web.core.windows.net/pull/28760/perf-test-northstar/VideoMinimalPerf.default_base.html) | 1.03:1
AccordionMinimalPerf.default | [85](https://fluentuipr.z22.web.core.windows.net/pull/28760/perf-test-northstar/AccordionMinimalPerf.default.html) | [83](https://fluentuipr.z22.web.core.windows.net/pull/28760/perf-test-northstar/AccordionMinimalPerf.default_base.html) | 1.02:1
DropdownManyItemsPerf.default | [384](https://fluentuipr.z22.web.core.windows.net/pull/28760/perf-test-northstar/DropdownManyItemsPerf.default.html) | [378](https://fluentuipr.z22.web.core.windows.net/pull/28760/perf-test-northstar/DropdownManyItemsPerf.default_base.html) | 1.02:1
HeaderMinimalPerf.default | [201](https://fluentuipr.z22.web.core.windows.net/pull/28760/perf-test-northstar/HeaderMinimalPerf.default.html) | [198](https://fluentuipr.z22.web.core.windows.net/pull/28760/perf-test-northstar/HeaderMinimalPerf.default_base.html) | 1.02:1
LoaderMinimalPerf.default | [191](https://fluentuipr.z22.web.core.windows.net/pull/28760/perf-test-northstar/LoaderMinimalPerf.default.html) | [187](https://fluentuipr.z22.web.core.windows.net/pull/28760/perf-test-northstar/LoaderMinimalPerf.default_base.html) | 1.02:1
PortalMinimalPerf.default | [82](https://fluentuipr.z22.web.core.windows.net/pull/28760/perf-test-northstar/PortalMinimalPerf.default.html) | [80](https://fluentuipr.z22.web.core.windows.net/pull/28760/perf-test-northstar/PortalMinimalPerf.default_base.html) | 1.02:1
SliderMinimalPerf.default | [749](https://fluentuipr.z22.web.core.windows.net/pull/28760/perf-test-northstar/SliderMinimalPerf.default.html) | [736](https://fluentuipr.z22.web.core.windows.net/pull/28760/perf-test-northstar/SliderMinimalPerf.default_base.html) | 1.02:1
StatusMinimalPerf.default | [406](https://fluentuipr.z22.web.core.windows.net/pull/28760/perf-test-northstar/StatusMinimalPerf.default.html) | [398](https://fluentuipr.z22.web.core.windows.net/pull/28760/perf-test-northstar/StatusMinimalPerf.default_base.html) | 1.02:1
IconMinimalPerf.default | [387](https://fluentuipr.z22.web.core.windows.net/pull/28760/perf-test-northstar/IconMinimalPerf.default.html) | [380](https://fluentuipr.z22.web.core.windows.net/pull/28760/perf-test-northstar/IconMinimalPerf.default_base.html) | 1.02:1
TextMinimalPerf.default | [188](https://fluentuipr.z22.web.core.windows.net/pull/28760/perf-test-northstar/TextMinimalPerf.default.html) | [185](https://fluentuipr.z22.web.core.windows.net/pull/28760/perf-test-northstar/TextMinimalPerf.default_base.html) | 1.02:1
AttachmentSlotsPerf.default | [638](https://fluentuipr.z22.web.core.windows.net/pull/28760/perf-test-northstar/AttachmentSlotsPerf.default.html) | [631](https://fluentuipr.z22.web.core.windows.net/pull/28760/perf-test-northstar/AttachmentSlotsPerf.default_base.html) | 1.01:1
DialogMinimalPerf.default | [433](https://fluentuipr.z22.web.core.windows.net/pull/28760/perf-test-northstar/DialogMinimalPerf.default.html) | [430](https://fluentuipr.z22.web.core.windows.net/pull/28760/perf-test-northstar/DialogMinimalPerf.default_base.html) | 1.01:1
FormMinimalPerf.default | [213](https://fluentuipr.z22.web.core.windows.net/pull/28760/perf-test-northstar/FormMinimalPerf.default.html) | [211](https://fluentuipr.z22.web.core.windows.net/pull/28760/perf-test-northstar/FormMinimalPerf.default_base.html) | 1.01:1
GridMinimalPerf.default | [186](https://fluentuipr.z22.web.core.windows.net/pull/28760/perf-test-northstar/GridMinimalPerf.default.html) | [184](https://fluentuipr.z22.web.core.windows.net/pull/28760/perf-test-northstar/GridMinimalPerf.default_base.html) | 1.01:1
LabelMinimalPerf.default | [216](https://fluentuipr.z22.web.core.windows.net/pull/28760/perf-test-northstar/LabelMinimalPerf.default.html) | [214](https://fluentuipr.z22.web.core.windows.net/pull/28760/perf-test-northstar/LabelMinimalPerf.default_base.html) | 1.01:1
MenuMinimalPerf.default | [493](https://fluentuipr.z22.web.core.windows.net/pull/28760/perf-test-northstar/MenuMinimalPerf.default.html) | [487](https://fluentuipr.z22.web.core.windows.net/pull/28760/perf-test-northstar/MenuMinimalPerf.default_base.html) | 1.01:1
RosterPerf.default | [1555](https://fluentuipr.z22.web.core.windows.net/pull/28760/perf-test-northstar/RosterPerf.default.html) | [1537](https://fluentuipr.z22.web.core.windows.net/pull/28760/perf-test-northstar/RosterPerf.default_base.html) | 1.01:1
PopupMinimalPerf.default | [346](https://fluentuipr.z22.web.core.windows.net/pull/28760/perf-test-northstar/PopupMinimalPerf.default.html) | [342](https://fluentuipr.z22.web.core.windows.net/pull/28760/perf-test-northstar/PopupMinimalPerf.default_base.html) | 1.01:1
ProviderMinimalPerf.default | [199](https://fluentuipr.z22.web.core.windows.net/pull/28760/perf-test-northstar/ProviderMinimalPerf.default.html) | [197](https://fluentuipr.z22.web.core.windows.net/pull/28760/perf-test-northstar/ProviderMinimalPerf.default_base.html) | 1.01:1
SplitButtonMinimalPerf.default | [2230](https://fluentuipr.z22.web.core.windows.net/pull/28760/perf-test-northstar/SplitButtonMinimalPerf.default.html) | [2199](https://fluentuipr.z22.web.core.windows.net/pull/28760/perf-test-northstar/SplitButtonMinimalPerf.default_base.html) | 1.01:1
TableManyItemsPerf.default | [1097](https://fluentuipr.z22.web.core.windows.net/pull/28760/perf-test-northstar/TableManyItemsPerf.default.html) | [1087](https://fluentuipr.z22.web.core.windows.net/pull/28760/perf-test-northstar/TableManyItemsPerf.default_base.html) | 1.01:1
ButtonOverridesMissPerf.default | [631](https://fluentuipr.z22.web.core.windows.net/pull/28760/perf-test-northstar/ButtonOverridesMissPerf.default.html) | [634](https://fluentuipr.z22.web.core.windows.net/pull/28760/perf-test-northstar/ButtonOverridesMissPerf.default_base.html) | 1:1
CheckboxMinimalPerf.default | [1131](https://fluentuipr.z22.web.core.windows.net/pull/28760/perf-test-northstar/CheckboxMinimalPerf.default.html) | [1129](https://fluentuipr.z22.web.core.windows.net/pull/28760/perf-test-northstar/CheckboxMinimalPerf.default_base.html) | 1:1
DatepickerMinimalPerf.default | [3510](https://fluentuipr.z22.web.core.windows.net/pull/28760/perf-test-northstar/DatepickerMinimalPerf.default.html) | [3524](https://fluentuipr.z22.web.core.windows.net/pull/28760/perf-test-northstar/DatepickerMinimalPerf.default_base.html) | 1:1
EmbedMinimalPerf.default | [1858](https://fluentuipr.z22.web.core.windows.net/pull/28760/perf-test-northstar/EmbedMinimalPerf.default.html) | [1858](https://fluentuipr.z22.web.core.windows.net/pull/28760/perf-test-northstar/EmbedMinimalPerf.default_base.html) | 1:1
TextAreaMinimalPerf.default | [274](https://fluentuipr.z22.web.core.windows.net/pull/28760/perf-test-northstar/TextAreaMinimalPerf.default.html) | [275](https://fluentuipr.z22.web.core.windows.net/pull/28760/perf-test-northstar/TextAreaMinimalPerf.default_base.html) | 1:1
ToolbarMinimalPerf.default | [525](https://fluentuipr.z22.web.core.windows.net/pull/28760/perf-test-northstar/ToolbarMinimalPerf.default.html) | [527](https://fluentuipr.z22.web.core.windows.net/pull/28760/perf-test-northstar/ToolbarMinimalPerf.default_base.html) | 1:1
TreeMinimalPerf.default | [468](https://fluentuipr.z22.web.core.windows.net/pull/28760/perf-test-northstar/TreeMinimalPerf.default.html) | [466](https://fluentuipr.z22.web.core.windows.net/pull/28760/perf-test-northstar/TreeMinimalPerf.default_base.html) | 1:1
CarouselMinimalPerf.default | [256](https://fluentuipr.z22.web.core.windows.net/pull/28760/perf-test-northstar/CarouselMinimalPerf.default.html) | [259](https://fluentuipr.z22.web.core.windows.net/pull/28760/perf-test-northstar/CarouselMinimalPerf.default_base.html) | 0.99:1
DropdownMinimalPerf.default | [1407](https://fluentuipr.z22.web.core.windows.net/pull/28760/perf-test-northstar/DropdownMinimalPerf.default.html) | [1425](https://fluentuipr.z22.web.core.windows.net/pull/28760/perf-test-northstar/DropdownMinimalPerf.default_base.html) | 0.99:1
InputMinimalPerf.default | [530](https://fluentuipr.z22.web.core.windows.net/pull/28760/perf-test-northstar/InputMinimalPerf.default.html) | [537](https://fluentuipr.z22.web.core.windows.net/pull/28760/perf-test-northstar/InputMinimalPerf.default_base.html) | 0.99:1
CustomToolbarPrototype.default | [1462](https://fluentuipr.z22.web.core.windows.net/pull/28760/perf-test-northstar/CustomToolbarPrototype.default.html) | [1483](https://fluentuipr.z22.web.core.windows.net/pull/28760/perf-test-northstar/CustomToolbarPrototype.default_base.html) | 0.99:1
AnimationMinimalPerf.default | [287](https://fluentuipr.z22.web.core.windows.net/pull/28760/perf-test-northstar/AnimationMinimalPerf.default.html) | [294](https://fluentuipr.z22.web.core.windows.net/pull/28760/perf-test-northstar/AnimationMinimalPerf.default_base.html) | 0.98:1
ListCommonPerf.default | [378](https://fluentuipr.z22.web.core.windows.net/pull/28760/perf-test-northstar/ListCommonPerf.default.html) | [385](https://fluentuipr.z22.web.core.windows.net/pull/28760/perf-test-northstar/ListCommonPerf.default_base.html) | 0.98:1
SkeletonMinimalPerf.default | [187](https://fluentuipr.z22.web.core.windows.net/pull/28760/perf-test-northstar/SkeletonMinimalPerf.default.html) | [191](https://fluentuipr.z22.web.core.windows.net/pull/28760/perf-test-northstar/SkeletonMinimalPerf.default_base.html) | 0.98:1
ButtonMinimalPerf.default | [82](https://fluentuipr.z22.web.core.windows.net/pull/28760/perf-test-northstar/ButtonMinimalPerf.default.html) | [85](https://fluentuipr.z22.web.core.windows.net/pull/28760/perf-test-northstar/ButtonMinimalPerf.default_base.html) | 0.96:1
CardMinimalPerf.default | [297](https://fluentuipr.z22.web.core.windows.net/pull/28760/perf-test-northstar/CardMinimalPerf.default.html) | [308](https://fluentuipr.z22.web.core.windows.net/pull/28760/perf-test-northstar/CardMinimalPerf.default_base.html) | 0.96:1
LayoutMinimalPerf.default | [189](https://fluentuipr.z22.web.core.windows.net/pull/28760/perf-test-northstar/LayoutMinimalPerf.default.html) | [198](https://fluentuipr.z22.web.core.windows.net/pull/28760/perf-test-northstar/LayoutMinimalPerf.default_base.html) | 0.95:1
TooltipMinimalPerf.default | [1241](https://fluentuipr.z22.web.core.windows.net/pull/28760/perf-test-northstar/TooltipMinimalPerf.default.html) | [1300](https://fluentuipr.z22.web.core.windows.net/pull/28760/perf-test-northstar/TooltipMinimalPerf.default_base.html) | 0.95:1

</details><div id="perfComment9424"/>

**Comment by fabricteam** [comment]
## 🕵 fluentuiv8 No visual regressions between this PR and main
<div id="vrtCommentfluent-ui_VRT_Pipeline_v8"/>

**Comment by fabricteam** [comment]
## 🕵 FluentUIV0 No visual regressions between this PR and main
<div id="vrtCommentfluent-ui_VRT_Pipeline_v0"/>

**Comment by fabricteam** [comment]
## 🕵 fluentuiv9 No visual regressions between this PR and main
<div id="vrtCommentfluent-ui_VRT_Pipeline_v9"/>

File: .github/CODEOWNERS [hunk 1]
```diff
@@ -239,6 +239,7 @@ packages/react-components/react-jsx-runtime @microsoft/teams-prg
 packages/react-components/react-toast @microsoft/teams-prg
```
Comment (fabricteam) [inline .github/CODEOWNERS:239]: ## [🕵 fluentuiv8 Open the Visual Regressions report to inspect the affected screenshots]( https://staticpagerelease.z16.web.core.windows.net/azurestaticapps/vr-approval/index.html?organization=uifabric&project=fabricpublic&prId=28760&commitId=fbd376caa7a6a4aab0b14093418d902d48e5782a&env=prod&clientType=FLUENTUI)


  <details><summary><b>MediaTile</b> 2 screenshots</summary>

|Image Name|Diff(in Pixels)| Image Type
|-|-|-|
|MediaTile.Media tile with two activity lines.default.chromium.png|0| Removed|
|MediaTile.Media tile with two activity lines.hover.chromium.png|0| Removed|

</details>




<div id="vrtCommentfluent-ui_VRT_Pipeline_v8"/>

**Review by jurokapsiar — APPROVED** [review]

**Review by ling1726 — APPROVED** [review]

**Pull request merged**